java/main/com/google/privacy/differentialprivacy/GaussianNoise.java

```diff
@@ -138,14 +144,39 @@ public ConfidenceInterval computeConfidenceInterval(
    */
   @Override
   public ConfidenceInterval computeConfidenceInterval(
-      long noisedX,
-      int l0Sensitivity,
-      long lInfSensitivity,
-      double epsilon,
-      Double delta,
-      double confidenceLevel) {
-    // TODO: Implement confidence interval computation.
-    throw new UnsupportedOperationException("Not implemented yet.");
+          long noisedX,
+          int l0Sensitivity,
+          long lInfSensitivity,
+          double epsilon,
+          Double delta,
+          double alpha) {
+    CheckConfidenceIntervalParameters(l0Sensitivity, lInfSensitivity, epsilon, delta, alpha);
+    double l2Sensitivity = lInfSensitivity * Math.sqrt((double) l0Sensitivity);
+    double sigma = getSigma(l2Sensitivity, epsilon, delta);
+    ConfidenceInterval confInt = computeConfidenceInterval((double) noisedX, sigma, alpha);
+    confInt =
+            ConfidenceInterval.create(
+                    Math.round(confInt.lowerBound()), Math.round(confInt.upperBound()));
```

Comment:
The formatting here looks weird. Could you move `ConfidenceInterval.create(` one line above?

(You could also create the confidenceInterval object and return in the same expression like we did for Go)